pages/component/inCart.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import {NextPage} from "next";
+import { Tab } from "../../layout/Tab"
+import { Box, Text, Image, Center, Button,    } from '@chakra-ui/react'
+import { useSelector, useDispatch } from "react-redux";
+import { CartState,deleteCart } from "../../redux/slice/cart";
+import { DeleteIcon } from '@chakra-ui/icons'
+import Link from "next/link";
+
+
+
+
+const inCart:NextPage = () => {
+	const dispatch = useDispatch();
+  const cartList:any = useSelector<CartState>((state:any) => state.cart.value);
+
+	//カート内の合計値取得(初期値０)
+	const total = cartList.reduce((sum: number,num: { price: number; }) => sum + num.price, 0 );
```

Comment:
再レンダーによる不要な再計算を防ぐため、`useMemo`でラップしてあげると良いです。

```suggestion
	const total = useMemo(() => cartList.reduce((sum: number,num: { price: number; }) => sum + num.price, 0 ), [cartList]);
```